Assets/static/scripts/application.js

```diff
@@ -15,11 +16,13 @@ class Page {
         }
         else if (this.state == Page.PAGE_MAIN) {
             url = "/pages?path=Main-page.html";
-            setTimeout(setup_messenger, 1000);
         }
-        else if (this.state = Page.PAGE_CONNECT_ERR) {
+        else if (this.state == Page.PAGE_CONNECT_ERR) {
```

Comment:
I noticed this mistake, while adding the URL for the new page content. I wondered if this was causing the issue I mentioned to you earlier about the server?